src/content/docs/intro/cli.mdx

```diff
@@ -17,11 +17,11 @@ The Vyuh CLI internally uses `pnpm` and `melos` to manage the packages for the
 schema and feature packages. This is to ensure that the packages are managed in
 a consistent way across all the projects.
 
-it's a good idea to have `pnpm` and `melos` installed globally. To read more
-about how to do it, refer to these links:
+For seamless functionality, ensure the following dependencies are installed and accessible in your shell path:
```

Comment:
you removed all the links to the tools. Please include it. Don't delete, instead merge the contents under one section